### ISSUE TITLE
Consider setting max size for images in overview

### ISSUE THREAD

**Issue by thibaultmol** [pietervdvn/MapComplete#2007]
![image](https://github.com/pietervdvn/MapComplete/assets/1010226/66b43387-7095-4865-8155-d54b1b132be8)
pin je punt for example has a large image, so maybe there should be a height limit for images? unless if you think it's okay, then leave it

**Comment by pietervdvn** [comment]
No, there should not be an image at all. I'm removing them from the overview.
